addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -7,13 +7,13 @@
             <div id="slideshow_sample" class="carousel slide" data-bs-ride="carousel" data-bs-interval="0" style="margin: 0 12px;">
                 <div class="carousel-inner" style="padding: 0;">
                     <div class="carousel-item active">
-                        <img class="img img-fluid d-block" src="/web/image/website.library_image_08" data-name="Image" data-index="0"/>
+                        <img class="img img-fluid d-block" src="/web/image/website.library_image_08" data-name="Image" data-index="0" alt=" "/>
```

Comment:
Note: After a discussion with ROLE, it has been decided to put `alt=" "` for those images. Indeed, those images will more than probably be replaced at the website creation. However, the advantage of  `alt=" "` over  `alt=""` (without a space) is that a screen reader will announce the image in the first case while it would skip it in the second case so even if an image has not been replaced (probably by mistake), it will still be announced.